business-central-parent/scripts/script.cli

```diff
@@ -0,0 +1,22 @@
+#/subsystem=security/security-domain=other:remove
+
+#/extension=org.wildfly.extension.elytron:add
+#/subsystem=elytron:add
+
+
+
+/subsystem=elytron/properties-realm=KIERealm:add(users-properties={path=application-users.properties, relative-to=jboss.server.config.dir, plain-text=true, digest-realm-name="KIEDomain"}, groups-properties={path=application-roles.properties, relative-to=jboss.server.config.dir})
```

Comment:
I need to change this to file system realm. Also the users and system properties should be added in this CLI file.